docker-entrypoint.sh

```diff
@@ -22,6 +22,8 @@ if [[ "$CREATE_ADMIN_USER" = "1" ]]; then
     fi
 fi
 
+python /app/manage.py generate_notifications_from_templates
```

Comment:
I think the problem with this being here would be that if more than one container is run multiple times (which happens for staging and production). Would moving this to `on_deploy.sh` work?